src/Common/src/System/FormattingHelpers.cs

```diff
@@ -2,10 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Globalization;
 
-namespace ILCompiler
+namespace System
```

Comment:
I rationalized this to myself since IConvertible and IFormattable are in the System namespace. Either way works, but I prefer to have things in some namespace.